File: src/main/java/frc/robot/util/Gains.java
package frc.robot.util;

import org.team997coders.spartanlib.helpers.PIDConstants;

public class Gains {
	public final double kP;
	public final double kI;
	public final double kD;
	public final double kF;
	
	public Gains(double _kP, double _kI, double _kD, double _kF){
		kP = _kP;
		kI = _kI;
		kD = _kD;
		kF = _kF;
	}

	public PIDConstants toPIDConstants() {
		return new PIDConstants(kP, kI, kD);
	}
}

File: src/main/kotlin/frc/robot/utils/Gains.kt
package frc.robot.utils

import org.team997coders.spartanlib.helpers.PIDConstants

class Gains {
  var P: Double = 0.0
  var I: Double = 0.0
  var D: Double = 0.0
  var F: Double = 0.0

  fun toPIDConstants(): PIDConstants {
    return PIDConstants(P, I, D)
  }
}